Surface_mesher/doc/Surface_mesher/dependencies

```diff
@@ -6,4 +6,5 @@ Circulator
 Stream_support
 Triangulation_2
 Triangulation_3
+Mesh_3
```

Comment:
That is surprising.

TODO: check why

Reply:
Note that this is doc/dependencies, not the include's dependencies.

Reply:
Oh, right! Let's cl;ose this conversation.